torch/_torch_docs.py

```diff
@@ -7754,3 +7754,126 @@ def merge_dicts(*dicts):
     >>> g_cpu.device
     device(type='cpu')
 """)
+
+add_docstr(torch.searchsorted,
+           r"""
+searchsorted(sorted_sequence, values, out_int32=False, right=False, out=None) -> Tensor
+
+Find the indices from the *innermost* dimension of :attr:`sorted_sequence` such that, if the 
+corresponding values in :attr:`values` were inserted before the indices, the order of the 
+corresponding *innermost* dimension within :attr:`sorted_sequence` would be preserved. 
+Return a new tensor, same size as :attr:`input`. The returned index satisfies the following rules: 
```

Comment:
nit: "Returns a new tensor with the same size as :attr:`input`". The comma in the middle of the sentence is weird